settings/test_with_coverage.py

```diff
@@ -0,0 +1,15 @@
+"""
+Django settings for running tests with coverage.
```

Comment:
The issue is that all tests were run with coverage, even when running a single test in the debugger. This change makes the behavior more like edx-platform, where we only run coverage when doing everything at once, i.e. on Travis.